{% extends 'layout.html' %}
{% block body %}
<h3>About plantiSMASH</h3>
<p>
  plantiSMASH allows the rapid genome-wide
  identification, annotation and analysis of specialized metabolite biosynthetic
  gene clusters (BGCs) across the plant kingdom. It is a specialized extension of the
  widely used <a href="http://antismash.secondarymetabolites.org/">antiSMASH</a>
  tool, tailored specifically to target plant genomes.
</p>

<p>
  plantiSMASH is powered by several open
  source tools: <a href='http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&PAGE_TYPE=BlastDocs&DOC_TYPE=Download'>NCBI BLAST+</a>,
  <a href="http://ab.inf.uni-tuebingen.de/software/diamond/">Diamond</a>,
  <a href='http://hmmer.janelia.org/'>HMMer 3</a>,
  <a href="http://www.drive5.com/muscle/">Muscle 3</a>,
  <a href="https://ccb.jhu.edu/software/glimmerhmm/">GlimmerHMM</a>,
  <a href="http://weizhongli-lab.org/cd-hit/">CD-HIT</a>,
  <a href='http://codeboje.de/pysvg/'>PySVG</a>,
  <a href='http://keith-wood.name/svg.html'>JQuery SVG</a>,
  <a href="https://datatables.net/">JQuery DataTables</a>,
  <a href="http://www.openscreen.cz/software/inchlib/home/">InCHlib</a>,
  <a href="http://visjs.org/">vis.js</a>, 
  <a href=" https://github.com/matsen/pplacer">pplacer</a> and
  <a href="https://huttenhower.sph.harvard.edu/graphlan/">GraPhlAn</a>. 
</p>

<p>
  plantiSMASH version 2.0 is the product of a collaborative effort between the
  <a href="http://www.wur.nl/en/Expertise-Services/Chair-groups/Plant-Sciences/Bioinformatics.htm">Bioinformatics Group</a> of Wageningen University &amp; Research,
  the <a href="https://www.jic.ac.uk/staff/anne-osbourn/">Department of Metabolic Biology</a> of the John Innes Centre, Norwich, 
  the <a href="https://plantscience.agri.huji.ac.il/people/guy-polturak">The Robert H. Smith Institute of Plant Sciences and Genetics in Agriculture</a> at The Hebrew University of Jerusalem, Herzl,
  and the <a href="https://jgi.doe.gov/">DOE Joint Genome Institute</a> at the Lawrence Berkeley National Labs.
</p>

<p>
  plantiSMASH version 2.0 is supported by 
  the Graduate School for Experimental Plant Sciences (EPS), Vidi Grant VI.Vidi.213.183 from The Netherlands Organization for Scientific Research (NWO). 
</p>

<p>
  plantiSMASH version 1.0 was the product of a collaborative effort between the
  <a href="http://www.wur.nl/en/Expertise-Services/Chair-groups/Plant-Sciences/Bioinformatics.htm">Bioinformatics Group</a> of Wageningen University &amp; Research,
  <a href="http://if.unila.ac.id/">the Department of Informatics Engineering</a> at the Lampung University,
  the <a href="http://www.biosustain.dtu.dk/">Novo Nordisk Foundation Center for Biosustainability</a> at the Technical University of Denmark,
  and the <a href="https://www.jic.ac.uk/staff/anne-osbourn/">Department of Metabolic Biology</a> of the John Innes Centre, Norwich.
</p>

<p>
  plantiSMASH version 1.0 was collaboratively supported by
  the Graduate School for Experimental Plant Sciences (EPS), VENI Grant
  (#863.15.002) from The Netherlands Organization for Scientific Research (NWO), a
  Novo Nordisk Foundation Grant, UK Biotechnological and Biological Sciences
  Research Council (BBSRC) Institute Strategic Programme Grant "Understanding and
  Exploiting Plant and Microbial Metabolism" (BB/J004561/1), the John Innes
  Foundation, the joint Engineering and Physical Sciences Research Council/
  BBSRC-funded OpenPlant Synthetic Biology Research Centre grant BB/L014130/1 and
  a National Institutes of Health Genome to Natural Products Network award U101GM110699.
</p>

<h3>How to cite</h3>
<p>If you have found plantiSMASH useful, please cite:</p>
<p>
  <b>plantiSMASH: automated identification, annotation and expression analysis of plant biosynthetic gene clusters.</b>
  <br />
  Satria A. Kautsar, Hernando G Suarez Duran, Kai Blin, Anne Osbourn &amp; Marnix H. Medema<br />
  <em>Nucleic Acids Research, 45(W1), W55-W63.</em> (2017) <a href="https://doi.org/10.1093/nar/gkx305">https://doi.org/10.1093/nar/gkx305</a><br>
</p>
{% endblock %}
