{% extends 'layout.html' %}
{% block body %}
<h3>Download plantiSMASH standalone</h3>
<div class="well well-small">
  Stand-alone versions of plantiSMASH are available through <b><a href="https://github.com/plantismash/plantismash">our GitHub page</a></b>.
</div>
<div class="tabbable tabs-left">
  <div class="tab-content">
    <div class="tab-pane active">
      <p>The current standalone release is <a href="https://github.com/plantismash/plantismash/tree/dev">plantiSMASH <strong>2.0-beta</strong></a>  (<em>December 16th, 2024</em>).</p>
    </div>

    <div>
      <h4>Install from source</h4>
      <p>
        First of all, download the plantiSMASH python source from our GitHub repo:<br />
        <a href="https://github.com/plantismash/plantismash/releases">https://github.com/plantismash/plantismash/releases</a>
      </p>
      <p>
        Then, make sure you have the following plantiSMASH dependencies installed:
        <ul>
            <li><a href="https://ccb.jhu.edu/software/glimmer/">glimmer</a> (version 3.02 tested)</li>
            <li><a href="https://ccb.jhu.edu/software/glimmerhmm/">GlimmerHMM</a> (version 3.0.4 tested)</li>
            <li><a href="http://hmmer.janelia.org/download.html">hmmer2</a> (version 2.3.2 tested, append a 2 to all hmmer2 executables to avoid conflict with hmmer3 executable names, like hmmalign -&gt; hmmalign2)</li>
            <li><a href="http://hmmer.janelia.org/download.html">hmmer3</a> (version 3.0 and 3.1b2 tested)</li>
            <li><a href="http://www.microbesonline.org/fasttree/#Install">fasttree</a> (version 2.1.7 tested)</li>
            <li><a href="https://github.com/bbuchfink/diamond">diamond</a> (&gt; 0.7.9 required, version 0.7.10 tested<sup id="fnref:1"><a class="footnote-ref" href="#fn:1" rel="footnote">1</a></sup>)</li>
            <li><a href="http://www.drive5.com/muscle/downloads.htm">muscle</a> (version 3.8.31 tested)</li>
            <li><a href="http://prodigal.ornl.gov/">prodigal</a> (version 2.6.1 tested)</li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/">NCBI blast+</a> (version 2.2.31 tested)</li>
            <li><a href="http://weizhongli-lab.org/cd-hit/download.php">CD-HIT</a> (version 4.6.6 tested)</li>
            <li><a href="http://tukaani.org/xz/">xz</a> development headers (version 5.1.1 tested)</li>
            <li><a href="http://xmlsoft.org">xml</a> development headers (version 2.9.1 tested)</li>
            <li>python (version 2.7 tested)</li>
            <li><a href="https://matsen.fhcrc.org/pplacer/">pplacer</a> (version 1.1 tested) </li>
            <li><a href="https://huttenhower.sph.harvard.edu/graphlan/">GraPhlAn</a> (version 1.1.3.1 tested) </li>
        </ul>
        <p>Create virtual conda enviroment.for installing the plantiSMASH python dependencies. </p>
        <pre><code class="bash hljs">mamba create -n plantismash python=2.7.18</code></pre>
        <p>Activate the conda environment.</p>
        <pre><code class="bash hljs">conda activate plantismash</code></pre>
        <p>Install dependencies by running<code>install_dependencies.py</code> : </p>
        <pre><code class="bash hljs">python install_dependencies.py</code></pre>
        <p>Last but not least, run <code>download_databases.py</code> to grab and prepare the databases needed for the analyses:</p>
        <pre><code class="bash hljs">python download_databases.py</code></pre>
      </p>
    </div>
    <div class="tab-pane" id="extras">
      <h4>Scripts that can aid in using antiSMASH:</h4>
      <dl class="dl-horizontal">
        <dt><a href="https://bitbucket.org/antismash/antismash2/downloads/antiSMASH2_pHMM_seed_alignments.zip">pHMM seed alignments</a>
        <dd>Seed alignments used for building pHMMs used in secondary metabolite cluster finding</dd>
        <dt><a href="https://bitbucket.org/antismash/antismash2/downloads/analysis_template.zip">analysis plug-in</a></dt>
        <dd>Template code for creating your own antismash 2.0 analysis plug-ins</dd>
        <dt><a href="https://bitbucket.org/antismash/antismash2/downloads/output_template.zip">output plug-in</a></dt>
        <dd>Template code for creating your own antismash 2.0 output plug-ins</dd>
        <dt><a href="https://bitbucket.org/antismash/antismash1/downloads/format_embl.zip">format_embl</a></dt>
        <dd>Script for preparing an EMBL file from a TXT file with gene annotations and a DNA sequence in FASTA format</dd>
        <dt><a href="https://bitbucket.org/antismash/antismash1/downloads/batch_antismash.zip">batch_antismash</a></dt>
        <dd>Script for batch-processing multiple files in antiSMASH version 1.x</dd>
      </dl>
    </div>
  </div>
</div>
{% endblock %}
{% block ready_function %}
  if (window.location.hash.length > 0) {
    $('.nav-tabs a[href=' + window.location.hash + ']').tab('show');
  }
{% endblock %}
